hw/ip/otbn/doc/_index.md

```diff
@@ -649,13 +668,70 @@ Before being stored in SRAM, the instruction word with the attached Integrity Pr
 The scrambling is reversed on a read.
 
 The ephemeral memory scrambling key and the nonce are provided by the [OTP block]({{<relref "/hw/ip/otp_ctrl/doc" >}}).
-They are set once when OTBN block is reset, and changed whenever a secure wipe of the instruction memory is performed.
-(TODO: Link to it once we have specified secure wipe.)
+They are set once when OTBN block is reset, and changed whenever a [secure wipe]({{<relref "#design-details-secure-wipe-imem">}}) of the instruction memory is performed.
 
 The Integrity Protection Code is checked on every memory read, even though the code remains attached to the data.
 A further check must be performed when the data is consumed.
 Detected integrity violations in the data memory raise a fatal `imem_error`.
 
+### Secure Wipe {#design-details-secure-wipe}
+
+The entire OTBN internal state including the instruction and data memories, can be securely deleted on demand from a host software.  
```

Comment:
We use the term "internal state" in this context as separate category next to instruction and data memories.

```suggestion
The entire OTBN state, including the contents of the instruction and data memories, can be securely deleted on demand from a host software.  
```